modules/spark/src/main/scala_2.11/spark-last/notebook/kernel/Repl.scala

```diff
@@ -103,11 +102,13 @@ class Repl(val compilerOpts: List[String], val jars:List[String]=Nil) extends Re
     //val i = new HackIMain(settings, stdout)
     loop = new org.apache.spark.repl.HackSparkILoop(stdout, outputDir)
 
-    jars.foreach { jar =>
+    val fps = jars.map{ jar =>
       import scala.tools.nsc.util.ClassPath
       val f = scala.tools.nsc.io.File(jar).normalize
-      loop.addedClasspath = ClassPath.join(loop.addedClasspath, f.path)
+      //loop.addedClasspath = ClassPath.join(loop.addedClasspath, f.path)
+      f.path
     }
+    settings.classpath.value=(classpath.distinct ::: fps).mkString(java.io.File.pathSeparator)
```

Comment:
I wonder if this sets driver classpath or both driver & executor?

Reply:
only driver, executor is done through the spark.jars conf